lib/results.js

```diff
@@ -104,6 +105,7 @@ Results.prototype._watch = function (t) {
     var write = function (s) { self._stream.queue(s); };
     t.once('prerun', function () {
         var premsg = '';
+        if (t._skip) self.skip ++;
```

Comment:
Wouldn't this line count a skipped `test` as a skipped `assertion`. Unlike in [line 122](https://github.com/substack/tape/pull/486/files#diff-79bfb5495a80c63110589a8e58db81f9R122) which counts skipped asserts, this line seems to include numbers that aren't consistent.

There is no equivalent to [Line 123](https://github.com/substack/tape/pull/486/files#diff-79bfb5495a80c63110589a8e58db81f9R123) in this `prerun` callback.

Reply:
Yes, this is just an update to an old PR meant to illustrate the problem you are mentioning: tests and assertions are mixed together. 

The reason the two are mixed together is that `tape` reports "assertions" as "tests", and doesn't count suites.
```
# tests 2
# pass  1
# fail  1
```

If we don't count suites, the only way to indicate that some assertions are skipped is to add one to tests because we don't know how many are actually skipped. That create the problem that you are mentioning.

The solution is probably to change the count and the output to be something like `node-tap`:

```
Suites:   1 failed, 1 passed, 1 skipped, 3 of 3 completed
Asserts:  2 failed, 3 passed, 2 skipped, of 7
```

My concern is not to break people with this change, so I will probably do this behind a flag. Once we agree on the output and the necessity of a flag, then the changes will be trivial. 

What's your take? If would be great if you could help me by testing the upcoming changes in your environment. :)